test/Garnet.test/RespSortedSetTests.cs

```diff
@@ -989,12 +990,59 @@ public void CheckSortedSetOperationsOnWrongTypeObjectSE()
             RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetRandomMember(keys[1]));
             // ZDIFF
             RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetCombine(SetOperation.Difference, keys));
+            // ZDIFFSTORE
+            RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetCombineAndStore(SetOperation.Difference, destionationKey, keys));
```

Comment:
typo: destinationKey (multiple instances in this file)

Reply:
Fixed